tests/contrib/test_postgresql.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+from typing import Iterable
+
+import testing.postgresql
+
+from flipper import PostgreSQLFeatureFlagStore
+from flipper.contrib.interface import FlagDoesNotExistError
+from flipper.contrib.storage import FeatureFlagStoreMeta
+from flipper.contrib.util.date import now
+
+Postgresql = testing.postgresql.PostgresqlFactory(cache_initialized_db=True)
+
+
+def tearDownModule(self):
+    Postgresql.clear_cache()
+
+
+class BaseTest(unittest.TestCase):
```

Comment:
These tests are rather slooow. I don't know that I care _that_ much because the project is small enough that it isn't a big deal. But... if there's anything you can do to make these run faster, it would be much appreciated. Thanks.

Reply:
I'll investigate :+1:

Reply:
I couldn't figure out a way to speed the tests here :(